tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -25,7 +25,7 @@ describe('Acceptance: brocfile-smoke-test', function() {
   this.timeout(500000);
 
   before(function() {
-    return createTestTargets(appName);
+    return createTestTargets(appName, { createESLintConfig: true });
```

Comment:
Can we name this option differently? I suspect that we will eventually want this to be `runDefaultBlueprints` or something like that.